config/odp-linux-generic.conf

```diff
@@ -18,6 +18,16 @@
 odp_implementation = "linux-generic"
 config_file_version = "0.0.1"
 
+# Internal shared memory allocator
+ishm: {
+	# ODP will try to reserve as many huge pages as the number indicated
+	# here, up to 64. Zero or a negative value means that no pages should
```

Comment:
Looks like the limit of 64 pages is removed in the next patch but this comment is not updated.